pkg/controller/machineset/machineset_controller.go

```diff
@@ -71,9 +71,8 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	mapper := msmapper{client: mgr.GetClient()}
 	err = c.Watch(&source.Kind{Type: &bmh.BareMetalHost{}},
-		&handler.EnqueueRequestsFromMapFunc{ToRequests: &mapper}, predicate.ResourceVersionChangedPredicate{})
+		handler.EnqueueRequestsFromMapFunc(mapperFunc), predicate.ResourceVersionChangedPredicate{})
```

Comment:
Could be wrong but I think go will allow you to do `EnqueueRequestsFromMapFunc(mapper.Map)` here.

Reply:
Fixed.